src/Stripe/Entities/StripeAccount.cs

```diff
@@ -29,7 +29,7 @@ public class StripeAccount : StripeObject
         public string DisplayName { get; set; }
 
         [JsonProperty("email")]
-        public string Email { get; set; }
+        public string Email { get; set; }
 
         [JsonProperty("statement_descriptor")]
         public string StatementDescriptor { get; set; }
```

Comment:
you may be able to remove your changes here entirely - I recently updated this file with stripe's docs